File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.js
function solution(n) {
    var answer = [];
    var x = 0;
    
    for (i = 0; i <= n; i++) {
        if (i % 2 != 0) {
            answer[x] = i;
            x++;
        }
    }
    
    
    
    return answer;
}

File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int[] answer = {};
        int x = 0;
        
        if (n % 2 == 0) {
            answer = new int[n/2];
        } else if (n % 2 != 0) {
            answer = new int[n/2+1];
        }
        
        for ( int i = 0; i <= n; i++) {
            if (i % 2 != 0) {
                answer[x] = i;
                x++;
            }
        }
        return answer;
    }
}